test/elixometer_test.exs

```diff
@@ -370,6 +370,18 @@ defmodule ElixometerTest do
 
   end
 
+  test "blacklisting subscriptions works" do
+    # Remove :median from the subscriptions
+    Application.put_env(:elixometer, :datapoints_blacklist, [:median])
+    update_histogram "uniquelittlefoobar", 42
+    wait_for_messages()
+    key = ["elixometer", "test", "histograms", "uniquelittlefoobar"]
+    datapoints = :exometer.info(key)[:datapoints]
+    subscriptions = Enum.filter(Reporter.subscriptions, fn x -> x == key end)
+
+    assert length(subscriptions) < length(datapoints)
```

Comment:
this assertion is a bit weak; Instead, it would be better if you asserted that median wasn't in the datapoints.

Reply:
I agree, I wasn't too happy with this and wanted to make it stronger, but because of how the reporter is implemented, we lose the datapoint information when we save subscriptions in the tests, see [here](https://github.com/pinterest/elixometer/blob/d1c85a31da3b450607a35bda2c8348fdfd4c93e4/test/support/test_reporter.ex#L35).

Do you think it's worth it for me to change that subscriptions to include the datapoints, and reinforce this test? I went for the smallest change possible but happy to extend the support `Reporter`

Reply:
I think that would be a nice improvement. It could be done separately and then merged first, and then this PR could use it.

Reply:
I added it to a PR into this branch in #93